frontend/src/styles/vars.scss

```diff
@@ -248,7 +248,7 @@ $_lifecycle_dormant: $_danger;
     --recording-highlight-event: var(--mark);
     --recording-highlight-event-dark: #946508;
 
-    // Z-indexes, synced with vars.ts
```

Comment:
There's no file vars.ts 🤔

Reply:
Looks like it was removed by @Twixes in https://github.com/PostHog/posthog/pull/17607

Reply:
Ah, yup, forgot about this line – no more z-indices in JS